spi/data-plane-selector/data-plane-selector-spi/src/test/java/org/eclipse/edc/connector/dataplane/selector/spi/instance/DataPlaneInstanceTest.java

```diff
@@ -86,6 +87,22 @@ void verifyCanHandle_withTransferType() throws MalformedURLException {
         assertThat(inst.canHandle(createAddress(srcType1), "notexist")).isFalse();
     }
 
+    @Test
+    void verifyCanProvisionDestination() throws MalformedURLException {
+        var provisionType = "provisionType";
+        var inst = DataPlaneInstance.Builder.newInstance()
+                .id("test-id")
+                .url(new URL("http://localhost:8234/some/path"))
+                .allowedSourceType("srcType")
+                .allowedTransferType("destType")
+                .destinationProvisionTypes(Set.of("provisionType"))
+                .build();
+
+        assertThat(inst.canProvisionDestination(null)).isTrue();
```

Comment:
if no destination is provided then there's no need to provision, this should return false.